src/main.ts

```diff
@@ -42,6 +47,9 @@ export async function activate(context: vscode.ExtensionContext): Promise<CSharp
     util.setExtensionPath(extension.extensionPath);
 
     const eventStream = new EventStream();
+    let optionStream = new BehaviorSubject<Options>(Options.Read(vscode));
+    let optionChangedStream = new Subject<void>();
+    optionChangedStream.debounceTime(1500).subscribe(() => { optionStream.next(Options.Read(vscode)); });
```

Comment:
Why do we need optionStream and optionChangedStream? Can't we set the debounce on optionStream and directly push it when we hear the event?